test/deployframework/local.go

```diff
@@ -0,0 +1,189 @@
+package deployframework
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/sirupsen/logrus"
+	"k8s.io/apimachinery/pkg/util/wait"
+)
+
+const (
+	ansibleRunnerPath                   = "/tmp/ansible-operator/runner"
+	meteringOperatorName                = "metering-ansible-operator"
+	meteringOperatorLogName             = "metering-operator.log"
+	reportingOperatorLogName            = "reporting-operator.log"
+	destKubeConfigPath                  = "/kubeconfig"
+	reportingOperatorTLSDirName         = "reporting-operator-tls"
+	runReportingOperatorLocalScriptName = "run-reporting-operator-local.sh"
+	cleanupScriptName                   = "run-test-cleanup.sh"
+
+	enableDebug        = true
+	disableOCPFeatures = false
+)
+
+// LocalCtx holds all the necessary information to run e2e tests locally
+type LocalCtx struct {
+	Namespace                      string
+	BasePath                       string
+	KubeConfigPath                 string
+	HackScriptPath                 string
+	MeteringOperatorImage          string
```

Comment:
nit: For consistency with the deployer.Config I think we should keep this separated as `meteringOperatorImageRepo` and `meteringOperatorImageTag` and then combine them when we call `exec.Command` in `RunMeteringOperatorLocal`.

Reply:
Updated.